lib/package-readme-view.js

```diff
@@ -45,13 +45,42 @@ function sanitize (html) {
     checkbox.setAttribute('disabled', true)
   }
 
+  let path = require('path')
+  let urlProtocolRegex = new RegExp("^(?:[a-z]+:)?\/\/", "i")
+
+  for (const image of temporaryContainer.querySelectorAll('img')) {
+    let imageSrc = image.getAttribute('src')
+
+    let changeSrc = true
```

Comment:
For extra clarity I'd rename this to `changeImageSrc`.

Reply:
Done in latest commit